agent.go

```diff
@@ -87,6 +87,7 @@ type sandbox struct {
 	sync.RWMutex
 
 	id              string
+	hostname        string
```

Comment:
These hostname changes seem unrelated? Maybe put them on a different commit with an explanation?

Reply:
It's related... The original code saves hostname to sandbox.id. I moved it to sandbox.hostname.

Reply:
Ah - right, thanks.